index.html

```diff
@@ -1,26 +1,59 @@
 <!DOCTYPE html>
```

Comment:
@daniellasheva I like the way you organized your HTML. You have 4 main classes: container, middle, Amanda, and yellow - all block elements. This is great. From there, it's just a matter of styling each section correctly.